Dockerfile

```diff
@@ -1,15 +1,18 @@
 FROM ruby:2.6.6-alpine as build-stage
 
 ENV RAILS_ENV=production
-ENV BUNDLE_APP_CONFIG="app/.bundle"
+ENV BUNDLE_WITHOUT="development:test"
+ENV BUNDLE_FROZEN="true"
 
```

Comment:
Using ENV variables for bundle instead of the command line arguments, which are on a deprecation course. Other option is having bundle config file which I'm not so keen to use in this context.